db/payment_icons.yml

```diff
@@ -2190,3 +2190,7 @@
   name: bancomat
   label: Bancomat
   group: other
+-
+name: orangemobilemoney
```

Comment:
Hi @tolu-paystack , the name here is not correctly formated. Kindly address it. 

Thanks

Reply:
Hey @adeniyiao, what's incorrect about the formatting? [An earlier comment](https://github.com/activemerchant/payment_icons/pull/1131#issuecomment-1932032566) said the name can't have dashes, spaces, or underscores, so I'm not sure how else to write it

Reply:
> Hey @adeniyiao, what's incorrect about the formatting? [An earlier comment](https://github.com/activemerchant/payment_icons/pull/1131#issuecomment-1932032566) said the name can't have dashes, spaces, or underscores, so I'm not sure how else to write it

Apologies if I wasn't clear. It's the position of the `name` which is not aligned in line 2194

<img width="329" alt="Screenshot 2024-04-03 at 10 00 59 AM" src="https://github.com/activemerchant/payment_icons/assets/97685764/744ca200-f95e-42f1-a602-cc5cbd0599a1">

Reply:
Got it, thanks @adeniyiao. This is fixed now